docs/docs/reference/debugger/debugger_vscode.md

```diff
@@ -58,11 +57,32 @@ Skips variables debugging instrumentation of code, making debugging less conveni
 Skipping instrumentation causes the debugger to be unable to inspect local variables.
 :::
 
+#### testName
+
+_String, optional._
+
+Test name (or substring) of the test function to debug. Is not required for the name to be exact as long as it's unequivocal
+For the debugger to run only one test function should match the name lookup
+
+:::note
+When provided the debugger will debug the matching function instead of the package `main` function.
+:::
+
+#### oracleResolver
+
+_String, optional._
+
+JSON RPC url to solve oracle calls
```

Comment:
```suggestion
JSON RPC URL to solve oracle calls.
```

Reply:
Let's add a little more explanation, it is true that this sets the target for oracle calls, but from the perspective of users/newcomers it is kind of obscure. I'd expand telling that this lets you interact with a TXE, which enables debugging tests that use aztec-contracts

Reply:
Mmmh, I'm not sure about that. I thought about it when writing this, but then I changed my mind because these docs are about noir-lang, not about aztec framework. 
I definitively think that there should be a "How to debug aztec contracts tests" guide, but I think that that guide should live in aztec's docs, not in noir's.